packages/orchestration/test/mocks/chain-info.js

```diff
@@ -0,0 +1,304 @@
+/**
+ * Generated using gRPC queries to local simulated chains
+ * See {@link https://github.com/Agoric/agoric-sdk/blob/43e0249502171e82460be7cf4bad8baf5db5aafd/packages/cosmic-proto/test/snapshots/test-chain-info.js.md | snapshots} and
+ * their {@link https://github.com/Agoric/agoric-sdk/blob/43e0249502171e82460be7cf4bad8baf5db5aafd/packages/cosmic-proto/test/snapshots/test-chain-info.js.md | source test}.
+ * // TODO check in some portion of above code
+ */
+export const mockChainInfo = {
+  agoric: {
+    icaParams: {
+      allowMessages: [
+        '/agoric.*',
+        '/ibc.*',
+        '/cosmos.[a-rt-z]*',
+        '/cosmos.s[a-su-z]*',
+      ],
+      hostEnabled: true,
+    },
+    icqParams: {
+      hostEnabled: false,
+    },
+    chainId: 'agoriclocal',
+    bondDenom: 'ubld',
+    denoms: [
+      {
+        denom: 'ubld',
+        native: true,
+      },
+      {
+        denom: 'uist',
+        native: true,
+      },
+      {
+        baseDenom: 'uion',
+        denom:
+          'ibc/F7E92EE59B5428793F3EF5C1A4CB2494F61A9D0C9A69469D02390714A1372E16',
+        path: 'transfer/channel-0',
+      },
+      {
+        baseDenom: 'uosmo',
+        denom:
+          'ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518',
+        path: 'transfer/channel-0',
```

Comment:
channel 0 goes to the cosmos hub, according to https://mapofzones.com/zones/agoric-3/peers?columnKey=ibcVolumeIn&period=24h

or is this about emerynet or something?